scripts/terra_service_banner.py

```diff
@@ -0,0 +1,98 @@
+import argparse
+from google.cloud import storage as gcs
+
+
+def convert_json_to_string(json, env):
+    """Convert json file to string if supplied with custom banner json file."""
+
+    # open json file and read contents to a string
+    with open(json, "r") as j:
+        custom_banner_string = j.read()
+
+    print(f'Starting custom banner upload to {env} Terra.')
+    update_service_banner(custom_banner_string, env)
+
+
+def update_service_banner(json, env):
+    """Push json to bucket in selected production environment."""
+
+    # create storage Client and assign destination bucket
+    storage_client = gcs.Client()
+
+    # set bucket and suitable_group based on env
+    if env == "prod":
+        bucket = storage_client.get_bucket("firecloud-alerts")
+        suitable_group = "fc-comms@firecloud.org"
+    else:
+        bucket = storage_client.get_bucket(f"firecloud-alerts-{env}")
+        suitable_group = (f"fc-comms@{env}.test.firecloud.org")
+
+    # define required filename (alerts.json) and upload json string to gcs
+    blob = bucket.blob("alerts.json")
+    blob.upload_from_string(json)
+
+    print(f'Setting permissions and security on banner json object in GCS location.')
+    # set metadata on json object (gsutil -m setmeta -r -h "Cache-Control:private, max-age=0, no-cache")
+    blob.cache_control = "private, max-age=0, no-cache"
+    blob.patch()
+
+    # set and save READ access for AllUsers on json object (gsutil ach ch -u AllUsers:R)
+    blob.acl.all().grant_read()
+    blob.acl.save()
+
+    # set and save OWNER access for suitable_group on json object (gsutil ach ch -g suitable_group:O)
+    blob.acl.group(suitable_group).grant_owner()
+    blob.acl.save()
+
+    print("Banner action complete.")
+
+
+def post_banner(env):
```

Comment:
I'd rename this something like `post_standard_banner` to make it clear that this is a basic template banner